.gitlab-ci.yml

```diff
@@ -79,6 +83,19 @@ NAME_JOB2:
                 - make test ARGS="-V"
 
 NAME_JOB3:
```

Comment:
At some point I would like to rename these into more human readable names....

Reply:
Yes, we should do that. I didn't know if we wanted to have a naming convention for these or something...